app/src/androidTest/java/org/mozilla/fenix/syncintegration/test_integration.py

```diff
@@ -2,7 +2,11 @@
 import sys
 
 
+def test_sync_account_settings(tps, gradlewbuild):
+    gradlewbuild.test('checkAccountSettings')
+
 def test_sync_history_from_desktop(tps, gradlewbuild):
+    os.chdir('app/src/androidTest/java/org/mozilla/fenix/syncintegration/')
     tps.run('test_history.js')
     gradlewbuild.test('checkHistoryFromDesktopTest')
```

Comment:
are these run manually on some kind of schedule?  If they're stable enough to run unattended and we want them to run on a fixed cycle, we can definitely add them as a taskgraph cron job.  thoughts?

Reply:
Until Jenkins was off these tests were running twice a day. But we can schedule the frequency as we want, both using Jenkins cron job or from the [Jenkins](https://github.com/mozilla-mobile/fenix/blob/master/Jenkinsfile#L3) file